src/RoleObjects/AccessibilityObject.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import * as createjs from 'createjs-module';
```

Comment:
Why was this imported? I don't think I see anything added below that would have required import, but I could very well be wrong! I do, however, see that we already had a createJS event created before this PR.

Reply:
@adagar If I remember, I got a Linting warning to Import `createjs`. But now, I tried removing the above import line and I dont get the warning. So, for now I will remove the import.

Reply:
- confirmed; we do need this here

Reply:
Previously we were able to build without needing this defined, or imported... I'm worried this might cause issues with some folks using the library that already have createjs defined in their environment.